src/redux/form/reducers/form.reducer.ts

```diff
@@ -0,0 +1,65 @@
+import { formAddFilm } from "./../actions/form.actions";
+import { FormState } from "./../form.models";
+import { createReducer } from "typesafe-actions";
+import { Film } from "../../../interfaces";
+import { formDeleteFilm, formEditFilm } from "../actions/form.actions";
+
+export const initialState: FormState = {
+  film: null,
+};
+
+export const formReducer = createReducer(initialState)
+  .handleAction(
+    formEditFilm.success,
+    (state: FormState, action: { payload: Film[] }) => ({
```

Comment:
why `Film[]` in payload. Check a possibility to type your all reducers  in app with Generics or other way, so you don't need to write it for each action. in past i did it somehow, and it was automatically typed.

Reply:
wrong payload in each action here